app/controllers/aws_texts_controller.rb

```diff
@@ -4,5 +4,6 @@ def index
   end
 
   def show
+    @aws_texts = AwsText.find(params[:id])
```

Comment:
すみません。前回気づいていなかったようですが，単数形になっていませんね。

```suggestion
    @aws_text = AwsText.find(params[:id])
```

Reply:
はい！
修正しました。